Makefile

```diff
@@ -395,7 +395,7 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=v0.2
+protoVer=latest
```

Comment:
ill change this to v0.3 once confirmed this pr for generation is correct 

cc @aaronc @fdymylja